### PR TITLE
cEP-0: coala Enhancements Proposals (CONTRIBUTORS, READ THIS)

### PR DESCRIPTION
**Pull request by sils**
CC @coala-analyzer/coala-contributors 

I know this is a bit political but I think it may helpus a lot defining our features better before we actually do them and also documenting them.

### CODE REVIEW

File: cEP-1.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+### Benevolent Limited Dictator (BLD)
+
+Members of the coala community have suggested the establishment of a Benevolent
+Limited Dictator (short: BLD).
+
+The BLD is a coala member with additional veto rights as described below. As
+per this cEP, Lasse Schuirmann is the BLD for coala.
+
+The community may at all times abandon the project and create a fork if the BDL
```
Comment (AbdealiLoKo) [inline cEP-1.md]: BDL -> BLD *

File: cEP-0.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+This cEP describes the introduction of cEP's (coala Enhancement Proposals) to
+the coala community.
+
+cEP's are used to propose and discuss new major features. After specifying
+features with a cEP they should be implemented as specified. Unspecified
+features should not be implemented.
+
+They are obviously inspired by
+[PEP1](https://www.python.org/dev/peps/pep-0001/0).
```
Comment (adtac) [inline cEP-0.md]: That link is 404. Try this: https://www.python.org/dev/peps/pep-0001/

Comment (Udayan12167) [inline cEP-0.md]: Yup doesn't work for me either

**Comment by sils** [comment]
Hey people, can you please up or downvote this comment as you like or dislike this PR?

@coala-analyzer/coala-contributors

File: cEP-1.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+### C Level Decision
+
+For a C level decision, a vote must be officially announced at the coala
+members channel and via the coala-members mailing list. The vote must be up for
+at least one week and is available to all members.
+
+For a decision to happen, a simple majority of all participating members must
+exist. If the same number of members is in favor as against the decision, the
+BDL may decide. The BLD also may block a decision with reasons provided within
```
Comment (tushar-rishav) [inline cEP-1.md]: BDL -> BLD

**Pull request merged**